src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -73,6 +81,13 @@ export class CalciteValueListItem {
     this.pickListItem.toggleSelected(coerce, emit);
   }
 
+  /*
+   ** @internal
+   */
+  @Method() async activateHandle() {
+    this.handleActivated = true;
```

Comment:
fixme: Do we need a method for this if its just a boolean? Can we handle with a property?

Reply:
My concern with a prop is that it might show up as part of the API and the user really shouldn't mess with this. Will try a prop with @internal to try and omit docs about it. I remember having issues with that before though.

Reply:
@internal prop seems to be working. Going forward with it.